backend/api/socApi.ts

```diff
@@ -2,7 +2,7 @@ export const socApiFactory = (client) => {
   const processResponse = () => (response) => response.body
   const get = (context, url) => client.get(context, url).then(processResponse())
   const getSocDetails = (context, offenderNo, socEnabled) =>
-    socEnabled ? get(context, `/api/offender/${offenderNo}`) : Promise.resolve(null)
+    socEnabled ? get(context, `/soc/nominal/nomsId/${offenderNo}`) : Promise.resolve(null)
```

Comment:
odd choice here - new apis should really be using kebab-case instead

Reply:
noted - will discuss